drizzlepac/runastrodriz.py

```diff
@@ -161,7 +160,7 @@
 
 
 # default marker for trailer files
-__trlmarker__ = '*** astrodrizzle Processing Version ' + __version__ + __version_date__ + '***\n'
+__trlmarker__ = '*** astrodrizzle Processing Version ' + __version__ + '***\n'
```

Comment:
```suggestion
__trlmarker__ = f'*** astrodrizzle Processing Version {__version__} ***\n'
```
(add a spece between version and `***`)